tools/release/query_pullrequests.py

```diff
@@ -0,0 +1,65 @@
+import openai
+import requests
+import argparse
+
+# python query_pullrequests.py --token <token> --start-date 2021-01-01 --end-date 2021-12-31 --temperature 0.5 --model-id text-davinci-002 --api-key <your OpenAI API key>
+parser = argparse.ArgumentParser(description='Query pull requests from GitHub API')

```

Comment:
I would put the logic in a main function then call it under `if name == __main__`